src/content/pages/backpacking-checklist.md

```diff
@@ -1,5 +1,110 @@
 ---
-title: "Backpacking checklist"
+title: "Backpacking Checklist"
 description: "this is meta description"
 draft: false
----
\ No newline at end of file
+--- ### Core Backpacking Needs
+Pack with padded hip belt
```

Comment:
This doesn't have a space in front of it like the rest of the lines below. Should there be a space?